apps/channels/datamodels.py

```diff
@@ -10,7 +10,7 @@
 
 logger = logging.getLogger("ocs.channels")
 
-AttachmentType = Literal["code_interpreter", "file_search"]
+AttachmentType = Literal["code_interpreter", "file_search", "experiment_files"]
```

Comment:
I don't think it really matters what this is called but I did prefer what you had before which was `ocs_attachments`